recipes/bashlex/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "bashlex" %}
+{% set version = "0.12" %}
+{% set sha256 = "94bcf8759f28286d4fe2bbc408fa50e9fb7c46fa7097eac5b7cd76d75ca9936d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - enum34
```

Comment:
`enum34` is not built for py3.6:

https://github.com/conda-forge/enum34-feedstock/blob/master/recipe/meta.yaml#L16

so you'll need a selector tag on this requirement, which also means that you can't use `noarch: python`.